tests/indent/dart/switch.dart

```diff
@@ -0,0 +1,9 @@
+void test() {
+  final a = 69;
+  switch (a) {
+    case 69:
+      break;
+    case 420:
+    default:
```

Comment:
Is this the convention for dart? or is it a de facto convention, or is it not?

Going out of the way to support this may affect others

Reply:
This is how the dart formatter is formatting and how VSCode did it when I tested.

Reply:
Here is an example from C# docs:

![image](https://user-images.githubusercontent.com/26344867/231521956-ae1a1592-0ab3-4725-a3d5-041b24d0e827.png)

Reply:
If so, what's breaking about this? 
I don't really see why this fails, probably will look at it again later

Reply:
Okay, saw why it's breaking.